src/mm-datepicker/mm-datepicker.scss

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+@import "_bourbon";
+@import "_color";
+@import "_mixins";
+
+:host {
+	position: relative;
+	display: inline-block;
+	vertical-align: middle;
+	font-size: 0em;
+	line-height: 0em;
+	@include clearfix();
+}
+
+.start, .end {
```

Comment:
Minimal disruption would be to go with ._mm_start, ._mm_end although it'll need to change in the markup as well